.travis.yml

```diff
@@ -4,14 +4,16 @@ env:
 language: go
 go:
   - 1.12.x
+  - 1.8.x
   - master
 before_install:
+  - go get github.com/go-playground/overalls
   - go get github.com/mattn/goveralls
 git:
   depth: 1
 script:
-  - go test -v -race ./... -coverprofile=profile.cov
-  - $GOPATH/bin/goveralls -coverprofile=profile.cov -service=travis-ci
+  - $GOPATH/bin/overalls -project=github.com/optimizely/go-sdk -covermode=count -debug
```

Comment:
should it be goveralls ?

Reply:
nope, its the library using to support for multiple packages.